webapp/src/Form/Type/RejudgingType.php

```diff
@@ -151,15 +151,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ->andWhere('t.enabled = 1')
                 ->addOrderBy('t.name');
 
-            $anyPublic = false;
+            $anyAllTeams = false;
```

Comment:
I think this variable name is not so clear. Better would be e.g. `$selectAllTeams`